### ISSUE TITLE
Autoconnect to multiple devices

### ISSUE THREAD

**Issue by naevtamarkus** [akexorcist/Android-BluetoothSPPLibrary#6]
First of all, thanks a lot for your library! This is exactly what I needed.

I am wondering... would you be willing to accept pull requests to modify the autoConnect method to allow a list of BT names as parameters (full match) besides a keyword (partial match)? 
You see, I already have the list of BT devices that I want to autoconnect to, and don't necesarily match a common keyword.

It's actually a very small change, but if you are not willing to accept PRs then I won't even bother.

Alternatively, one could still use keywords but add $ and ^ to the end/begin. What I would still be missing is accept a list of keywords instead of a string.
Thanks!

**Comment by akexorcist** [comment]
Yes, I will accept to pull request from another if it make this library more complete. :D
